cfme/cloud/instance.py

```diff
@@ -242,6 +247,10 @@ def load_details(self, refresh=False):
             if refresh:
                 toolbar.refresh()
 
+    def open_edit(self):
+        self.load_details(refresh=True)
+        toolbar.select("Configuration", "Edit this Instance")
```

Comment:
I was thinking about it, but still, it would have to still keep this method in order to work polymorphically in the test, because I read the contents of the Select with parent VM.